README.md

```diff
@@ -114,36 +114,36 @@ For more usage details, see our detailed [documentation](#documentation).
 
 Benchmarks are run in GitHub Actions, and the tables below are updated with every code change. **Values >1.00x represent how much _slower_ each gem’s memoized value retrieval is than the latest commit of `MemoWise`**, according to [`benchmark-ips`](https://github.com/evanphx/benchmark-ips) (2.9.2).
 
-Results using Ruby 3.1.1:
+Results using Ruby 3.1.2:
```

Comment:
Was there supposed to be a change in the Github action or Gemfile or .ruby-version or something for this?